src/Umbraco.Web.UI.Client/src/packages/tiptap/components/toolbar/tiptap-toolbar-button.element.ts

```diff
@@ -48,7 +48,7 @@ export class UmbTiptapToolbarButtonElement extends UmbLitElement {
 				@click=${() => (this.api && this.editor ? this.api.execute(this.editor) : null)}>
 				${when(
 					this.manifest?.meta.icon,
-					() => html`<umb-icon name=${this.manifest!.meta.icon}></umb-icon>`,
+					(icon) => html`<umb-icon name=${icon}></umb-icon>`,
```

Comment:
Wow, did not know you could do this!

Reply:
I know! I only recently found out. Saves us putting `!` bangs in there. 😅